Mathlib/Data/Complex/Module.lean

```diff
@@ -202,7 +206,8 @@ end Complex
 
 /- Register as an instance (with low priority) the fact that a complex vector space is also a real
 vector space. -/
-instance (priority := 900) Module.complexToReal (E : Type _) [AddCommGroup E] [Module ℂ E] :
+noncomputable instance (priority := 900) Module.complexToReal
```

Comment:
`noncomputable` here also worth a porting note: I assume it's the same reason.

Note that Lean4 no longer flags unecessary `noncomputable`s as far as I'm aware, hence the need for notes.